Can't set cursor on Mac, when using LWJGL2 and AWT

#### Issue details
It is currently not possible to set the system cursor or any cursor on a Mac, when embedding a lwjgl canvas in an awt application.
#### Reproduction steps/code
```java
package com.badlogic.gdx.tests.lwjgl;

import java.awt.BorderLayout;
import java.awt.Container;

import javax.swing.JFrame;
import javax.swing.SwingUtilities;

import com.badlogic.gdx.backends.lwjgl.LwjglCanvas;
import com.badlogic.gdx.tests.CursorTest;

public class LwjglCanvasTest extends JFrame {
	LwjglCanvas canvas;

	public LwjglCanvasTest () {
		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);

		Container container = getContentPane();
		canvas = new LwjglCanvas(new CursorTest());

		container.add(canvas.getCanvas(), BorderLayout.CENTER);

		pack();
		setVisible(true);
		setSize(800, 480);
	}

	public static void main (String[] args) {
		SwingUtilities.invokeLater(new Runnable() {
			@Override
			public void run () {
				new LwjglCanvasTest();
			}
		});
	}
}
```
#### Version of LibGDX and/or relevant dependencies
LibGdx 1.9.2 Lwjgl-Backend

#### Stacktrace
```java
com.badlogic.gdx.utils.GdxRuntimeException: Could not create cursor image.
	at com.badlogic.gdx.backends.lwjgl.LwjglCursor.<init>(LwjglCursor.java:80)
	at com.badlogic.gdx.backends.lwjgl.LwjglGraphics.newCursor(LwjglGraphics.java:619)
	at com.badlogic.gdx.tests.CursorTest.create(CursorTest.java:44)
	at com.badlogic.gdx.backends.lwjgl.LwjglCanvas.create(LwjglCanvas.java:192)
	at com.badlogic.gdx.backends.lwjgl.LwjglCanvas$1$1.run(LwjglCanvas.java:86)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: org.lwjgl.LWJGLException: Native cursors not supported
	at org.lwjgl.input.Cursor.<init>(Cursor.java:88)
	at com.badlogic.gdx.backends.lwjgl.LwjglCursor.<init>(LwjglCursor.java:78)
	... 18 more
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [x] MacOS

#### Possible Fix:
It should be possible to directly set the cursor on the canvas using awt's tools.

--- a/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglCursor.java
+++ b/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglCursor.java
@@ -5,16 +5,20 @@ import java.nio.ByteOrder;
 import java.nio.IntBuffer;
 
 import org.lwjgl.LWJGLException;
-import org.lwjgl.input.Mouse;
 
+import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Cursor;
 import com.badlogic.gdx.graphics.Pixmap;
 import com.badlogic.gdx.utils.GdxRuntimeException;
+import com.badlogic.gdx.utils.SharedLibraryLoader;
 
 public class LwjglCursor implements Cursor {
 	org.lwjgl.input.Cursor lwjglCursor = null;
 
 	public LwjglCursor (Pixmap pixmap, int xHotspot, int yHotspot) {
+		if (((LwjglGraphics)Gdx.graphics).canvas != null && SharedLibraryLoader.isMac) {
+			return;
+		}
 		try {
 			if (pixmap == null) {
 				lwjglCursor = null;

--- a/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphics.java
+++ b/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphics.java
@@ -37,6 +37,7 @@ import com.badlogic.gdx.graphics.Pixmap;
 import com.badlogic.gdx.graphics.Pixmap.Format;
 import com.badlogic.gdx.utils.Array;
 import com.badlogic.gdx.utils.GdxRuntimeException;
+import com.badlogic.gdx.utils.SharedLibraryLoader;
 
 /** An implementation of the {@link Graphics} interface based on Lwjgl.
  * @author mzechner */
@@ -620,6 +621,9 @@ public class LwjglGraphics implements Graphics {
 
 	@Override
 	public void setCursor (com.badlogic.gdx.graphics.Cursor cursor) {
+		if (canvas != null && SharedLibraryLoader.isMac) {
+			return;
+		}
 		try {
 			Mouse.setNativeCursor(((LwjglCursor)cursor).lwjglCursor);
 		} catch (LWJGLException e) {
@@ -629,6 +633,9 @@ public class LwjglGraphics implements Graphics {
 	
 	@Override
 	public void setSystemCursor (SystemCursor systemCursor) {
+		if (canvas != null && SharedLibraryLoader.isMac) {
+			return;
+		}
 		try {
 			Mouse.setNativeCursor(null);
 		} catch (LWJGLException e) {

